pyomo/repn/util.py

```diff
@@ -65,15 +65,14 @@ class FileDeterminism(enum.IntEnum):
     SORT_SYMBOLS = 30
 
     # We will define __str__ and __format__ so that behavior in python
-    # 3.11 is consistent with 3.7 - 3.10.
+    # 3.11 is consistent with 3.8 - 3.10.
```

Comment:
This is a bit of a silly change.  The previous comment was not incorrect, and implied a longer-term behavior that we were preserving.